src/omlt/gbt/gbt_formulation.py

```diff
@@ -1,20 +1,38 @@
 import numpy as np
 import pyomo.environ as pe
 
-from omlt.formulation import _PyomoFormulation
+from omlt.formulation import _PyomoFormulation, _setup_scaled_inputs_outputs
 from omlt.gbt.model import GradientBoostedTreeModel
 
 
-class BigMFormulation(_PyomoFormulation):
+class GBTBigMFormulation(_PyomoFormulation):
+    def __init__(self, gbt_model):
+        super().__init__()
+        self.model_definition = gbt_model
+
+    @property
+    def input_indexes(self):
+        # TODO: Francesco - is this correct? Does GBT support multi-dimensional inputs?
+        return list(range(self.model_definition.n_inputs))
```

Comment:
I don't think GBT support multi-dimensional inputs. All libraries mention that the input for training must be a matrix with shape `(num_samples, num_features)`.